keyboards/gray_studio/space65/space65.c

```diff
@@ -38,6 +38,11 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
 
 void led_set_kb(uint8_t usb_led) {
 	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+  if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
```

Comment:
```suggestion
  if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
```

Reply:
You can also do
```suggestion
  (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK) ? writePinHigh : writePinLow)(E6);
```
if you like that better.

Reply:
Yeah, I agree. Just change it to use `IS_LED_ON` then.

Reply:
Yikes, I've never considered using the ternary operator to select between function pointers. I'm pretty sure I don't like it :)

Reply:
It's an idiom I came across in some embedded projects. It's like `++i %= len` in C++: at first you're like wtf, but then as you see it more and more you get used to it and it becomes fairly readable. But yes, it's not very noob friendly, and I wouldn't quite call it kernel quality code either.